pkg/i2gw/notifications/config.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package notifications
+
+import (
+	"os"
+
+	"github.com/jedib0t/go-pretty/v6/table"
+	"github.com/jedib0t/go-pretty/v6/text"
+)
+
+const (
+	bgColor = text.BgBlack
+
+	whiteColor   = text.FgHiWhite
+	magentaColor = text.FgHiMagenta
+	redColor     = text.FgRed
+)
+
+var (
+	notificationColumnNumber      = 2
+	maxWidthforNotificationColumn = 100
+
+	tableStyle    table.Style = table.StyleRounded
+	rowSeperation             = true
+	textAlignment text.Align  = text.AlignCenter
+)
+
+func newTableConfig() table.Writer {
+	t := table.NewWriter()
+
+	t.SetOutputMirror(os.Stdout)
```

Comment:
consider removing this. Then when you call `CreateNotificationTables` assign it to a `table` variable and return the table from `ToGatewayApiResources` 

Then I'd use it somewhere [here](https://github.com/kubernetes-sigs/ingress2gateway/blob/2333bec43b76da886ec521b3a9aea5bf28652a22/cmd/print.go#L92) to print it. I think thats a bit cleaner.